<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transfert</class>
 <widget class="QDialog" name="Transfert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>transfert</string>
  </property>
  <widget class="QComboBox" name="comboBox_article">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_montant">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>25</height>
    </size>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OK_Button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>570</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="Montant_label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>190</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom Transfert</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_nom_dotation">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_description">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>DÃ©tails de la transaction :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Montant</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>1er Article</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Transfert</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>2eme Article</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_article_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Montant_label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>240</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>160</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="article_radio_1">
      <property name="text">
       <string>article</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="programme_radio_1">
      <property name="text">
       <string>programme :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>160</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="article_radio_2">
      <property name="text">
       <string>article</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="programme_radio_2">
      <property name="text">
       <string>programme :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
